src/assets/styles/styles/custom/_button.scss

```diff
@@ -0,0 +1,33 @@
+.btn {
+  background: none;
+  border: none;
+  padding: 5px 20px 5px;
+  border-radius: 20px;
+  font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande", "Lucida Sans", Arial,
+    sans-serif;
+  font-size: 15px;
+  font-weight: medium;
+  text-align: center;
+  transition: background-color 0.5s, color 0.5s;
+  &:hover {
+    cursor: pointer;
+    background-color: gold;
```

Comment:
Pomyślałam, że highlight po najechaniu na button to dobry pomysł. Nie musi być złoty, chociaż moim zdaniem całkiem spoko w kontekście całościowego końcowego efektu wizualnego. Byłby ciekawym akcentem kolorystycznym. No, ale nie jestem jakoś specjalnie przywiązana do tej linijki jeżeli uznacie, że za głośno krzyczy w oczy, można go zmienić na bardziej klasyczny steelblue czy coś w tym rodzaju ;)

Reply:
Moim zdaniem lepiej jednak to by wyglądało bez tego podświetlenia, jakaś delikatna zmiana tła moim zdaniem byłaby lepsza. Jednak ekspertem od UI to ja nie jestem i zostawimy tę kwestię naszej grafik. Poruszymy ten temat na następnym spotkaniu z nią. Na razie może to zostać w ten sposób.